inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -85,6 +87,7 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
 
     manager.register_pass<ngraph::pass::ConstantFolding>();
     manager.register_pass<ngraph::pass::StridedSliceOptimization>(); // depends on CF
+    manager.register_pass<ngraph::pass::ConvertGather1ToGather7>(); // all transformations use Gather-7
```

Comment:
Move this transformation right after `RemoveFilteringBoxesBySize` pass

Reply:
changed to the solution with `GatherBase` class. No need to run this transformation in `common_optimizations` anymore